pages/policy.html

```diff
@@ -5,6 +5,24 @@
 ---
 
 <div id="section-full">
+    <div class="section-header">
+        <h3>Gameplay streaming, Let's Play content, and Monetization Statement</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>The core game of OpenTTD is released under the GNU General Public License (GPL) version 2, as are the OpenGFX, OpenSFX, and OpenMSX baseset content packs. The GPL permits use of the software for any purpose including commercial purposes.</p>
+            <p>Following is a statement of intention from the developers. This is not a legal statement. If any questions arise from this statement, please consult the license text of the involved software and game data.</p>
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset content packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the content is permitted without limitations.</p>
+            <p>Note that this statement does not imply any endorsement from any of the developers of OpenTTD or the baseset content towards any content creators. Any views expressed by content creators in content using the OpenTTD game and baseset content packs are purely those of the content creator.</p>
```

Comment:
This sentence is a bit hard to read. It says `content` a lot, but not always in the same context. And in general it feels a bit meh vs the other sentences. But I am also having a hard time coming up with an alternative, so I am not really useful here, sorry :(

Reply:
I changed it to "game data" when that's what it means.